libpcan/package.xml

```diff
@@ -1,26 +1,21 @@
-<package>
+<?xml version="1.0"?>
+<package format="2">
   <name>libpcan</name>
   <version>0.6.1</version>
   <description>This package wraps the libpcan to use it as a ros dependency</description>
-  <!-- <maintainer>Florian Weisshardt</maintainer> -->
 
   <license>LGPL</license>
 
   <url type="website">http://p103112.typo3server.info/fileadmin/media/linux/index.htm</url>
-  <!-- <url type="bugtracker"></url> -->
 
   <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
   <author>Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
+
   <build_depend>libpopt-dev</build_depend>
+  <build_depend>linux-headers-generic</build_depend>
   <build_depend>mk</build_depend>
-  <build_depend>rosbuild</build_depend>
   <build_depend>rospack</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>linux-headers-generic</build_depend>
```

Comment:
Did we? I thought this was the "safe way to go", if we are unsure.
And as `mk` and `rospack` are definitely no `exec_depend`s and are not exported (as they are not used in the code, but only to download and build the library), `build_depend` should be fine, I guess.

On the other hand, it does not hurt, so I'll make them a `depend` for now...